life/original/08-chapter-8.adoc

```diff
@@ -94,4 +94,4 @@ and cheerfully lay down their life and die with him to the world;
 they and none else shall rise with him in the newness of life;
 and ascend with him to his Father, by him to be glorified with that glory,
 which he had with his Father before the world began.
-Unto which kingdom, God Almighty conduct you, through this earthly pilgrimage: Amen.`"
```

Comment:
I wasn't sure if this should be changed....it's like this in the original, but just doesn't seem right.